distance_next_goal: 1.4
angle_next_goal: 0.5
check_rate: 15.0